README.md

```diff
@@ -1,35 +1,42 @@
-# OpenPower Firmware Build Environment
+# OpenPOWER Firmware Build Environment
```

Comment:
Also, we still use the all-caps version for the processor names - so we've got POWER8, POWER9, OpenPOWER... and Power Architecture. Only slightly confusing.